__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+afterAll(() => {
+  db.end();
```

Comment:
since you have block quoted db.end() you want to return it here (it is a promise)